### ISSUE TITLE
Prevent page scroll on Up/Down arrow keys

### ISSUE THREAD

**Issue by mtomov** [TarekRaafat/autoComplete.js#58]
Hi, your library looks, and works great! Thank you for putting the effort to make it.

The only thing I found not to work as expected was that the Arrow Up & Down keys would scroll the page when moving up and down the result list, which is indeed the _default_ webpage behaviour. 

I think the default behaviour needs somehow be prevented, and for the moment, I added this piece to capture the events at the _wrapper_ element:

```javascript
    // prevent page scroll
    document.querySelector('#autoComplete_wrapper').onkeydown = function(e) {
      if (e.key === 'ArrowUp' || e.key === 'ArrowDown') {
        e.view.event.preventDefault()
      }
    }
```

I think the problem is not apparent in your demo, because there's no scrollable content on the page.

Thank you!

**Comment by TarekRaafat** [comment]
Hello @mtomov,

Thank you so much, I'm really glad you like it and find it useful! :)

Thank you for reporting that behavior, I'm aware of it since a while it happens because of the navigation logic is focus based which leads to scrolling the page most of the time when the `focus` moves depend on the page height as you've mentioned.

I'm currently working on an entirely new navigation logic that's `css` based to prevent that undesirable behavior and build up for more future improvements as well (Inline autoComplete, fixed cursor in input field while navigation, etc..).

Hopefully, this new navigation logic will come up soon because I'm giving it a high priority.


Have a good day, cheers! :)

**Comment by mtomov** [comment]
Thank you! Appreciate the response!

**Comment by braco** [comment]
@TarekRaafat do you have an ETA on this feature?

**Comment by TarekRaafat** [comment]
Hello @braco,

Hopefully, version 7 would be coming out on the 30th of August and it's going to be pretty loaded with new stuff already but I'll do my best to implement the new navigation logic as well.

Have a good day, cheers! :)

**Comment by godot882** [comment]
Hi @TarekRaafat any update on this? I've just started to use autoComplete.js and this is something I've just came across. I would rather wait for version 7 instead of registering an event listener. Thanks in advance

**Comment by TarekRaafat** [comment]
Hello All,

I've just published the new [v7.1](https://github.com/TarekRaafat/autoComplete.js/releases/tag/v7.1.0) that has the new navigation logic.

> [v7.1.0](https://github.com/TarekRaafat/autoComplete.js/releases/tag/v7.1.0) :sparkles:
- New improved Navigation logic (Thanks 👍 @mtomov)
- `shadowRoot` API support `[Removed]`
- Enhanced `resList.navigation` API data feedback
- Styling `[Changed]`
- Major code Refactor & Optimizations
  - Faster performance
  - Reduced `autoComplete.js` weight

[Github](https://github.com/TarekRaafat/autoComplete.js/releases/tag/v7.1.0), [NPM](https://www.npmjs.com/package/@tarekraafat/autocomplete.js), [CDN](https://www.jsdelivr.com/package/npm/@tarekraafat/autocomplete.js)

Kindly check it and let me know your thoughts.

Have a good day, Cheers! :)

**Comment by mtomov** [comment]
Thank you @TarekRaafat !
